modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/ClusterManagementGroupManager.java

```diff
@@ -185,6 +189,8 @@ public ClusterManagementGroupManager(
         this.clusterIdStore = clusterIdChanger;
         this.raftGroupOptionsConfigurer = raftGroupOptionsConfigurer;
 
+        clusterResetStorage = new ClusterResetStorage(vault);
```

Comment:
Not related to this particular PR: we have ` ClusterIdHolder clusterIdChanger` which is assigned to `this.clusterIdStore`. So is it a store, a changer or a holder? =)

Reply:
It's a store. I renamed parameters and changed their type